pynequa/models.py

```diff
@@ -94,11 +95,14 @@ def generate_payload(self, **kwargs) -> Dict:
         This method generates payload for
         AdvancedParams.
         """
-        payload = {
-            self.col_name: self.col_value,
-            "value": self.value,
-            "operator": self.operator
-        }
+        payload = {}
+        # To prevent payloads with empty values
+        if self.col_name and self.col_value:
+            payload[self.col_name] = self.col_value
+        if self.value:
```

Comment:
```suggestion
        if self.value and self.operator:
```

The documentation says: `This parameter must be used with the operator parameter`

Reply:
Reference: https://doc.sinequa.com/en.sinequa-es.v11/Content/en.sinequa-es.devDoc.webservice.rest-search.html#query-adv-param